core/device.cpp

```diff
@@ -453,9 +456,12 @@ void Device::get_param_float_async(const std::string &name,
 
 MavlinkCommands::Result Device::set_flight_mode(FlightMode device_mode)
 {
-    uint8_t flag_safety_armed = is_armed() ? MAV_MODE_FLAG_SAFETY_ARMED : 0;
+    const uint8_t flag_safety_armed = is_armed() ? MAV_MODE_FLAG_SAFETY_ARMED : 0;
```

Comment:
Don't we need `Device::is_hitl_enabled()` for plugins in the future ?
If you think its useful, then its good time to add it.

Reply:
I would add it once we need it.